packages/lesswrong/components/tagging/TagSubforumPage.tsx

```diff
@@ -100,7 +115,15 @@ export const TagSubforumPage = ({ classes, user }: { classes: ClassesType; user:
           />
         </AnalyticsContext>
       </SingleColumnSection>
-      <div className={classes.columnSection}></div>
+      <div className={classes.columnSection}>
+        {tag?.tableOfContents?.html &&
+          <div className={classes.aside}>
```

Comment:
I believe this changes the behavior of what happens when there's no description.

Reply:
Fixed in https://github.com/ForumMagnum/ForumMagnum/pull/5678/commits/565d04f6530c8d8a745d7265d4f693350e518a89.